README.md

```diff
@@ -231,6 +231,26 @@ We also introduce three clustering metrics: the silhouette coefficient, the Davi
 | Davies-Bouldin index                    | Measures how similar an object is to its own cluster compared to other clusters. The value ranges from -1 to 1. A high value indicates appropriate clustering.   |
 | Calinski-Harabasz index                    | 1. Between-Cluster Dispersion: It measures how far the clusters are from each other. For good clustering, this should be as large as possible. 2.Within-Cluster Dispersion: It measures how compact the clusters are internally. For good clustering, this should be as small as possible. |
 
+#### 6. Priority index
+
+CohortFinder provides a visualization tool for the **priority index**, which guides the annotation workflow by ranking images based on their importance or informativeness.
```

Comment:
Should highlight that the priority index is the main feature here. 

The figure is really only to demonstrate the priority index at work. It's a visualization, but it isn't a tool.

Is that also your understanding?